front/src/modules/pipeline/editable-field/components/ProbabilityEditableField.tsx

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useState } from 'react';
+
+import { EditableField } from '@/ui/editable-field/components/EditableField';
+import { FieldContext } from '@/ui/editable-field/states/FieldContext';
+import { RecoilScope } from '@/ui/recoil-scope/components/RecoilScope';
+
+import { ProbabilityFieldEditMode } from './ProbabilityFieldEditMode';
+
+type OwnProps = {
+  icon?: React.ReactNode;
+  value: number | null | undefined;
+  onSubmit?: (newValue: number) => void;
+};
+
+export function ProbabilityEditableField({ icon, value, onSubmit }: OwnProps) {
+  const [internalValue, setInternalValue] = useState(value);
+
+  useEffect(() => {
```

Comment:
@emilienchvt Not sure if we really need to have an internal value, as it's not doing anything more than being update by the props value, we can just have `onChange` and `onCancel` props